technic_worldgen/nodes.lua

```diff
@@ -1,152 +1,191 @@
 	description = S("Uranium Ore"),
 	tiles = {"default_stone.png^technic_mineral_uranium.png"},
 	is_ground_content = true,
-	groups = {cracky=3, radioactive=1},
-	sounds = default.node_sound_stone_defaults(),
+	groups = {cracky=3, radioactive=1, pickaxey=1},
+	_mcl_hardness = 0.8,
+	_mcl_blast_resistance = 1,
+	sounds = technic.sounds.node_sound_stone_defaults(),
 	drop = "technic:uranium_lump",
 })
 
 minetest.register_node(":technic:mineral_chromium", {
 	description = S("Chromium Ore"),
 	tiles = {"default_stone.png^technic_mineral_chromium.png"},
 	is_ground_content = true,
-	groups = {cracky=3},
-	sounds = default.node_sound_stone_defaults(),
+	groups = {cracky=3, pickaxey=1},
+	_mcl_hardness = 0.8,
+	_mcl_blast_resistance = 1,
+	sounds = technic.sounds.node_sound_stone_defaults(),
 	drop = "technic:chromium_lump",
 })
 
 minetest.register_node(":technic:mineral_zinc", {
 	description = S("Zinc Ore"),
 	tiles = {"default_stone.png^technic_mineral_zinc.png"},
 	is_ground_content = true,
-	groups = {cracky=3},
-	sounds = default.node_sound_stone_defaults(),
+	groups = {cracky=3, pickaxey=1},
+	_mcl_hardness = 0.8,
+	_mcl_blast_resistance = 1,
+	sounds = technic.sounds.node_sound_stone_defaults(),
 	drop = "technic:zinc_lump",
 })
 
 minetest.register_node(":technic:mineral_lead", {
 	description = S("Lead Ore"),
 	tiles = {"default_stone.png^technic_mineral_lead.png"},
 	is_ground_content = true,
-	groups = {cracky=3},
-	sounds = default.node_sound_stone_defaults(),
+	groups = {cracky=3, pickaxey=1},
+	_mcl_hardness = 0.8,
+	_mcl_blast_resistance = 1,
+	sounds = technic.sounds.node_sound_stone_defaults(),
 	drop = "technic:lead_lump",
 })
 
 minetest.register_node(":technic:mineral_sulfur", {
 	description = S("Sulfur Ore"),
 	tiles = {"default_stone.png^technic_mineral_sulfur.png"},
 	is_ground_content = true,
-	groups = {cracky=3},
-	sounds = default.node_sound_stone_defaults(),
+	groups = {cracky=3, pickaxey=1},
+	_mcl_hardness = 0.8,
+	_mcl_blast_resistance = 1,
+	sounds = technic.sounds.node_sound_stone_defaults(),
 	drop = "technic:sulfur_lump",
 })
 
-minetest.register_node(":technic:granite", {
+if has_mcl then
+   minetest.register_alias("technic:granite", "mcl_core:granite")
+   minetest.register_alias("technic:granite_bricks", "mcl_core:granite_smooth")
+else
+   minetest.register_node(":technic:granite", {
 	description = S("Granite"),
 	tiles = {"technic_granite.png"},
 	is_ground_content = true,
 	groups = {cracky=1},
-	sounds = default.node_sound_stone_defaults(),
-})
+	sounds = technic.sounds.node_sound_stone_defaults(),
+   })
 
-minetest.register_node(":technic:granite_bricks", {
+   minetest.register_node(":technic:granite_bricks", {
 	description = S("Granite Bricks"),
 	tiles = {"technic_granite_bricks.png"},
 	is_ground_content = true,
 	groups = {cracky=1},
-	sounds = default.node_sound_stone_defaults(),
-})
+	sounds = technic.sounds.node_sound_stone_defaults(),
+   })
+end
```

Comment:
Please use tabs for indentation, not spaces.

If you can, you should change your editor's default indentation to tabs, as that is the preferred style.

Reply:
my editor use tabs by default, is configured to use tabs. A tab is 2 spaces in my config now, before it was 3 spaces.

Reply:
That sounds like the tab width setting. A tab is a different character, not multiple spaces.

Reply:
I think that Ive fixed it now